cft-templates/ec2-jupyterLab.yml

```
@@ -160,8 +165,17 @@ Resources:
                 echo "fetching token and starting jupyterlab"
                 `docker-compose up -d`                           
                 sleep 5
-                instance_id=$(curl -s "http://169.254.169.254/latest/meta-data/instance-id")
-                instance_region=$(curl -s "http://169.254.169.254/latest/meta-data/placement/region")
+                # Get the session token
+                TOKEN=$(curl -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 21600")
+                
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)